ethpm_types/contract_type.py

```diff
@@ -227,25 +227,40 @@ def __modify_schema__(cls, field_schema):
     def __get_validators__(cls):
         yield cls.validate_uri
         yield cls.validate_genesis_hash
+        yield cls.validate_block_hash
 
     @classmethod
     def validate_uri(cls, uri):
-        assert uri.startswith("blockchain://"), "Must use 'blockchain' protocol"
-        assert (
-            len(uri.replace("blockchain://", "").split("/")) == 3
-        ), "must be referenced via <genesis_hash>/block/<block_hash>"
+        if not uri.startswith("blockchain://"):
+            raise ValueError("Must use 'blockchain' protocol")
+
+        if len(uri.replace("blockchain://", "").split("/")) != 3:
+            raise ValueError("must be referenced via <genesis_hash>/block/<block_hash>")
```

Comment:
Nitpick: standardize the returned error strings
```suggestion
        if not uri.startswith("blockchain://"):
            raise ValueError("Must use 'blockchain' protocol.")

        if len(uri.replace("blockchain://", "").split("/")) != 3:
            raise ValueError("Must be referenced via <genesis_hash>/block/<block_hash>.")
```
or 
```suggestion
        if not uri.startswith("blockchain://"):
            raise ValueError("must use 'blockchain' protocol")

        if len(uri.replace("blockchain://", "").split("/")) != 3:
            raise ValueError("must be referenced via <genesis_hash>/block/<block_hash>")
```